client/my-sites/people/people-invites/index.jsx

```diff
@@ -129,15 +129,35 @@ class PeopleInvites extends React.PureComponent {
 			return requesting ? this.renderPlaceholder() : this.renderEmptyContent();
 		}
 
+		const pendingLabel = translate(
+			'You have a pending invite for %(numberPeople)d user',
```

Comment:
:information_source: This string is probably long enough that it doesn't need a **context**. Are you sure you don't want to use a translator comment instead?